src/mpi/errhan/errnames.txt

```diff
@@ -949,6 +949,11 @@ is too big (> MPIU_SHMW_GHND_SZ)
 **set_thread_affinity:Failed to set the async thread affinity
 **set_thread_affinity %d:Failed to set the async thread affinity to the logical processor [%d]
 
+## MPIX_Stream
+**ch3nostream:Stream is not supported in ch3.
+**ch4nostream:No streams available. Configure --enable-thread-cs=per-vci and --with-ch4-max-vcis=# to enable streams.
+**outofstream:No streams available. Use MPIR_CVAR_CH4_RESERVE_VCIS to reserve the number of streams can be allocated.
```

Comment:
nitpick: its not obvious that vci is what is meant by "stream" in these error messages

Reply:
What do you suggest?

Reply:
I guess the issue is that this error message is meant for callers of `MPIX_Stream_create`? In that case it makes sense, its just odd to try to allocate X and get an error message that there are no more Y.

Reply:
I see. Right, the message is meant for `MPIX_Stream_create`. Only the lower layer has the details on what is wrong, so we have to craft the message there.